dynawo/sources/Models/Modelica/Dynawo/Types.mo

```diff
@@ -65,5 +65,9 @@ package Types "Standard types for electrical variables"
   // Percent
   type Percent = Real (unit = "100") "Percent quantity";
 
+  type AngularVelocityPu = PerUnit;
```

Comment:
Could you put it with the other per unit types ?

Reply:
Yes. Moreover, should I put the pu types in alphabetical order ?

Reply:
Not necesary